src/AutoRest.CSharp/Common/Input/InputTypes/Examples/InputOperationExample.cs

```diff
@@ -6,5 +6,5 @@
 namespace AutoRest.CSharp.Common.Input.Examples
 {
     // TODO -- currently we do not need the responses. In the case that in the future we need to handle the responses in examples, we need to add a new class InputExampleResponse and a new property here
-    internal record InputOperationExample(InputOperation Operation, IReadOnlyList<InputParameterExample> Parameters, string? Name = null, string? ExampleFilePath = null);
+    internal record InputOperationExample(string Name, string? Description, string FilePath, IReadOnlyList<InputParameterExample> Parameters);
```

Comment:
why does `InputOperationExample` not have an operation?
And why is it better to set it directly in `MockTestCase`?

Reply:
I see why you are doing this now, align with TCGC output.

Reply:
yeah it is because TCGC does not have it.
It might be bad practice to have it from the json's perspective because the InputOperation holds examples and examples hold the operation itself.